pilot/lib/pilot/build_manager.rb

```diff
@@ -128,7 +128,7 @@ def list(options)
 
     def update_beta_app_meta(options, build)
       # App Store Connect API build id
-      build_id = build.find_app_store_connect_build["id"]
+      build_id = build.id
```

Comment:
Looks like @joshdholtz fixed this in a different way in https://github.com/fastlane/fastlane/commit/7b2d9ab0847d02f60966f2084120efc447f41d04#diff-fb9d332b3e3b1f3dfb3c5968c4f0c218L131